### PR TITLE
Create ui-c8y-1020-0-0-navigator-node-property-is-now-correctly-changed-after-dashboard-restore.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by ensu

Ticket: [MTM-58690](https://cumulocity.atlassian.net/browse/MTM-58690)
Original PR: [6098](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6098)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-navigator-node-property-is-now-correctly-changed-after-dashboard-restore.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: navigator node property is now correctly changed after dashboard restore.
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-navigator-node-property-is-now-correctly-changed-after-dashboard-restore.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Navigator node property correctly changed after dashboard restore
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-navigator-node-property-is-now-correctly-changed-after-dashboard-restore.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58690
+version: 1020.0.0
+---
+navigator node property is now correctly changed after dashboard restore.
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-navigator-node-property-is-now-correctly-changed-after-dashboard-restore.md:17]: The Claude AI suggested to use the following body:
```suggestion
In the past, restoring a dashboard did not properly update the node property of the navigator, leading to inconsistencies. This has now been fixed. The navigator node property is correctly changed after a dashboard restore, ensuring the restored dashboard matches the expected state. This improvement provides a more reliable and consistent user experience when restoring dashboards.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by eniosultan — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-navigator-node-property-is-now-correctly-changed-after-dashboard-restore.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58690
+version: 1020.0.0
+---
+In the past, restoring a dashboard did not properly update the node property of the navigator, leading to inconsistencies. This has now been fixed. The navigator node property is correctly changed after a dashboard restore, ensuring the restored dashboard matches the expected state. This improvement provides a more reliable and consistent user experience when restoring dashboards.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-navigator-node-property-is-now-correctly-changed-after-dashboard-restore.md:17]:
```suggestion
In the past, restoring a dashboard did not properly update the node property of the navigator, leading to inconsistencies. This has now been fixed. The navigator node property is correctly changed after a dashboard restore, ensuring the restored dashboard matches the expected state.
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**